package/MDAnalysis/lib/distances.py

```diff
@@ -389,6 +389,118 @@ def self_distance_array(reference, box=None, result=None, backend="serial"):
 
     return distances
 
+def capped_distance(reference, configuration, max_cutoff, min_cutoff = 0, box = None, method = None):
```

Comment:
the spaces for the kwargs don't follow pep8 here.  something like flake8 etc can sort this out